docs/userguide/modules/creating-service-bindings/pages/custom-path-injection.adoc

```diff
@@ -132,3 +132,77 @@ metadata:
 spec:
     secret: binding-request-72ddc0c540ab3a290e138726940591debf14c581
 ....
+
+== Workload resource mapping
+
+The specification also provides a way of defining exactly where binding data needs to be projected
+into, which may work in situations where the above methods won't work. This is done by defining
+`ClusterWorkloadResourceMapping.servicebindings.io` resources, which define where binding data gets
+projected for a workload type.  This works for both CoreOS and specification service binding
+resources.
+
+The schema of these resources looks like the following:
+[source,yaml]
+....
+apiVersion: servicebinding.io/v1alpha3
+kind: ClusterWorkloadResourceMapping
+metadata:
+  name: # string <1>
+  generation: # int64, defined by the Kubernetes control plane
+  ...
+spec:
+  versions: # required, at least one entry must be defined
+  - version: # string <2>
+    annotations: # string (Fixed JSONPath), optional <3>
+    containers: # optional
+    - path: # string (JSONPath) <4>
+      name: # string (Fixed JSONPath), optional <5>
+      env: # string (Fixed JSONPath), optional <6>
+      volumeMounts: # string (Fixed JSONPath), optional <7>
+    volumes: # string (Fixed JSONPath), optional <8>
+....
+<1> the name of this resource (must be qualified as `plural.group`, i.e. `cronjobs.batch` for
+CronJobs)
+<2> the version of the resource we're mapping (can match all unspecified versions with the wildcard "*")
+<3> where annotations live with a fixed JSONPath (default: `.spec.template.spec.annotations`)
+<4> where containers live with a JSONPath (if no entries are defined, SBO defaults to two paths:
+`.spec.template.spec.containers` and `.spec.template.spec.initContainers` with all other fields
+set as their default).
+<5> corresponds to the `.name` field in a pod. Defaults to `.name`.
+<6> corresponds to the `.env` environment variables. Defaults to `.env`.
+<7> corresponds to where the volume mounts of a pod lives. Defaults to `.volumeMounts`.
+<8> where volumes live with a fixed JSONPath (default: `.spec.template.spec.volumes`)
+
+In this context, a "Fixed JSONPath" is a subset of the JSONPath grammar that accepts only two
+operations: field lookup (i.e. `.spec.template`) and indexing (i.e. `.spec['template']`).  All other
+operations are forbidden.
+
+Most of these fields are optional, and SBO assumes PodSpec-compatible defaults when not specified.
+SBO requires that each of these fields is semantically equivalent to the corresponding field in a Pod
+deployment; failing to maintain this may result in unexpected behavior.
+
+Using this schema, we can write an example mapping for `CronJob.batch/v1` resources:
+
+[source,yaml]
+....
```

Comment:
```suggestion
----
```

Reply:
On consideration, I don't understand why this change is necessary.  The other yaml listings in this file use `....` as delimiters, so why do I need to use `----` here?

Reply:
This is the asciidoc convention we follow in downstream docs. As we take some good amount of content from upstream for downstream purposes, it is good to practice the same thing here so both are in sync.

Reply:
Fixed for new instances.